### ISSUE TITLE
hue light becomes unreachable after turning it off

### ISSUE THREAD

**Issue by nattlip** [athombv/homey#674]
**What is your Homey version (Settings → System)?**

0.8.36

**What did you try to achieve?**

i achieved turning hue light off, but i sa after that the horrible unreachable favicon. 

_Please include detailed steps how we can reproduce the issue_

turn off a hue light. and then try to turn it on. it doesnt happen all the times , and hebecomes reachable again after some time.

**What was the result?**

...

**What did you expect as result?**

...

**Any other remarks**

...

**Comment by WeeJeWel** [comment]
It only happens when the bridge reports the light is unreachable, so I can't really do anything about that I'm afraid.

**Comment by nattlip** [comment]
that is not the issue, in my hue android app all lights are present

**Comment by JPe2** [comment]
After 0.8.36 was installed, I also had to uninstall Hue (and also Somfy) and reinstall them, to get them work again.

**Comment by sossienl** [comment]
Same problem here. Hue lights were all not available from Homey, but in the app no problems. The upside of it all is that my Homey was very responsive again (after weeks of serious issues). With the HUE lights not being available, it seems that all the other parts of Homey are working great.

But Homey not able to control my lights is a huge gap in the system, so I tried to fix this problem by removing the app. With that all the reports and HUE devices that were already in Homey. 

I reinstalled the app, but now Homey is unresponsive again. No way to get into or control Homey (website, app, voice). Back to the where all problems started I'm afraid. Last firmware that worked for me, including all HUE devices was 0.8.34.
